etl/steps/data/garden/climate/2023-12-20/surface_temperature.countries.json

```diff
@@ -194,5 +194,6 @@
   "S\u00e3o Tom\u00e9 and Pr\u00edncipe": "Sao Tome and Principe",
   "U.S. Virgin Islands (US)": "United States Virgin Islands",
   "Venezuela, Republica Bolivariana de": "Venezuela",
-  "Yemen, Republic of": "Yemen"
+  "Yemen, Republic of": "Yemen",
+  "Global": "Global"
```

Comment:
I don't mind using "Global" within the scripts, if you think that avoids confusion. But in the resulting garden and grapher datasets, as well as in the charts, it would make more sense to use "World" because that's what we use everywhere. We use it for example in [the climate change explorer](https://ourworldindata.org/explorers/climate-change). It would be confusing to have sometimes "World" and sometimes "Global". I don't think we have stated anywhere that "World" means land (although for most datasets that's what it implies).

Reply:
hey @pabloarosado! thanks for taking a look at the pull request! I've now implemented the changes but need to re-run the snapshot data upload (because of the directory change) which is taking a while because of the Copernicus API queues during the day. It shouldn't change anything in the rest of the code though. Could you please take a look at the changes you've asked for and approve if you are happy with them so I can merge the pull request as soon I get the data downloaded (which will likely be in the evening when the API queues are shorter) without having to wait till tmrw.

Reply:
also, I think using "Global" is better for the reasons I've explained above but since it would be more effort for you to change it then for me I have renamed it to "World" now.

Reply:
p.s. the unit tests will be failing because I am still uploading the dataset to snapshot but I will check that everything is ok before merging.